### DISCUSSION TITLE
Data format request

### DISCUSSION THREAD

**Discussion by damienmaguire** [collin80/SavvyCAN#336] [Q&A]
So I have a bunch of logs in the format:
19056 ID: 287 DLC: 8  0x27 0x10 0x27 0x10 0x27 0x10 0x00 0x00
Is there any way for Savvycan to accept this?

**Comment by collin80** [comment]
Sure, I can easily add a format like that to the loaders. However, what would I call it? What kind of file format is that?

**Comment by damienmaguire** [comment]
Not sure of the format but Ive seen it a few times now and have a bunch of logs of this type. I think its generated by an arduino logging example used with an mcp2515. I manage by hacking it into the ID/Data format for Savvycan but sure would be useful to have native support. At least to me:)
